Dockerfile

```diff
@@ -1,38 +1,47 @@
-
-
-FROM ubuntu:21.04 as build
+FROM ubuntu:20.04 as build
```

Comment:
why the drop down to 20.04? does a bump up to 22.04 not work?

Reply:
Provides greater compatibility using 20.04. AppImage documentation says should use the lowest supported LTS version should be used. It is always up to the developer, but I usually package one LTS version older. `20.04`